src/analysis/processing/qgsalgorithmsplitvectorlayer.cpp

```diff
@@ -65,6 +65,9 @@ void QgsSplitVectorLayerAlgorithm::initAlgorithm( const QVariantMap & )
   addParameter( new QgsProcessingParameterFeatureSource( QStringLiteral( "INPUT" ), QObject::tr( "Input layer" ) ) );
   addParameter( new QgsProcessingParameterField( QStringLiteral( "FIELD" ), QObject::tr( "Unique ID field" ),
                 QVariant(), QStringLiteral( "INPUT" ) ) );
+  std::unique_ptr< QgsProcessingParameterBoolean > prefixFieldParam = std::make_unique< QgsProcessingParameterBoolean >( QStringLiteral( "PREFIX_FIELD" ),
+      QObject::tr( "Add field prefix to layer names" ), true );
```

Comment:
What do you think about

```suggestion
      QObject::tr( "Add field prefix to file names" ), true );
```

Reply:
Makes sense to me!